packages/shared/friends/utils.ts

```diff
@@ -22,7 +52,27 @@ export function getUserIdFromMatrix(userId: string) {
  * from: '0x1111ada11111'
  * to: '@0x1111ada11111:decentraland.org'
  * */
-export function getMatrixIdFromUser(userId: string) {
+function getMatrixIdFromUser(userId: string) {
   // TODO check how'd be the correct way to do this
   return '@' + userId.toString() + ':decentraland.org'
 }
+
+/**
+ *
+ */
+function updateSocialInfo(socialData: SocialData) {
```

Comment:
this belongs to the sagas file, utils are just for functions that don't have any side effects